uwblueprint/extend-a-family
# add onboarding work/team member page

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Setup-Onboarding-Task-d6e0cdeb69cf4bbf93a33c7302370550)



## Implementation description
* Implemented a page to view all team members and the ability to add a John Cena team member on button click



## Steps to test
1. Go to /team-members in the frontend
2. Check if the table renders all team-members from your database if there are any in there
3. Click on the Add John Cena button and see if it adds in a John Cena with role PM
4. Refresh page to see if the changes persist



## What should reviewers focus on?
* 


## Checklist
